lib/oban/queue/basic_engine.ex

```diff
@@ -382,7 +382,7 @@ defmodule Oban.Queue.BasicEngine do
   defp since_period(query, period) do
     since = DateTime.add(utc_now(), period * -1, :second)
 
-    where(query, [j], j.inserted_at > ^since)
+    where(query, [j], j.inserted_at >= ^since)
```

Comment:
This change seems entirely valid, but I'm curious how frequently it comes up outside of a test environment 🤔